satwatch/docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: satwatch_web_1
+      APP_PORT: 8080
+  web:
+    image: ghcr.io/jpcummins/sat.watch:1.1.5@sha256:8fbdd04b01d1ab8fa5626a908622a1dc36fea1c93c24eb24b8213e0bb2a94b5a
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      DATABASE_URL: "postgresql://satwatch:satwatch@db:5432/satwatch?sslmode=disable"
+      ELECTRUM_HOST: $APP_ELECTRS_NODE_IP
+      ELECTRUM_PORT: $APP_ELECTRS_NODE_PORT
+      URL: "http://$DEVICE_DOMAIN_NAME:3883"
+      ZMQ_HOST: $APP_BITCOIN_NODE_IP
+      ZMQ_PORT: $APP_BITCOIN_ZMQ_RAWTX_PORT
+      RPCHOST: "http://$APP_BITCOIN_NODE_IP:$APP_BITCOIN_RPC_PORT"
+      RPCUSER: $APP_BITCOIN_RPC_USER
+      RPCPASSWORD: $APP_BITCOIN_RPC_PASS
+    user: "1000:1000"
+    depends_on:
+      - db
```

Wait for db to be healthy.
```suggestion
    depends_on:
      db:
        condition: service_healthy
```